src/app.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
+import readline from 'node:readline';
+import { generateRandomNumber } from './modules/generateRandomNumber.js';
+import { checkIsValidUserInput } from './modules/checkIsValidUserInput.js';
+import { getBullsAndCows } from './modules/getBullsAndCows.js';
 
-// Write your code here
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+const hiddenNumber = generateRandomNumber();
+
+const requestANumber = () => {
+  // console.log(hiddenNumber);
+
+  rl.question('Write your number? ', (userValue) => {
+    const isValidUserInput = checkIsValidUserInput(userValue);
```

Comment:
Ensure that the `checkIsValidUserInput` function correctly validates the input. The `checkIsUniqueNumbers` function currently returns true prematurely, which can lead to incorrect validation results. Make sure it checks all digits for uniqueness before returning true.